### PR TITLE
MES-4884 - Added Office Page for Home Tests

### PR DESCRIPTION
**Pull request by TomBillingtonUK**
## Description
- Adds Office Page for home tests
- Changes the page so that it no longer uses a getTestData Selector

## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: src/pages/office/cat-home-test/__tests__/office.cat-be.page.spec.ts [hunk 1]
```diff
@@ -0,0 +1,353 @@
+import { ComponentFixture, async, TestBed, fakeAsync, tick } from '@angular/core/testing';
```
Comment (appleabs) [inline src/pages/office/cat-home-test/__tests__/office.cat-be.page.spec.ts]: file is still cat-be

**Review by appleabs — APPROVED** [review]

File: src/pages/office/cat-home-test/__tests__/office.cat-be.page.spec.ts (hunk 1 above)
Comment (TomBillingtonUK) [reply src/pages/office/cat-home-test/__tests__/office.cat-be.page.spec.ts]: Nice find - Fixed :)

**Review by mgoldson — APPROVED** [review]
Looks good but what story does the removal of independent Driving and Route get removed?

**Comment by TomBillingtonUK** [comment]
I would assume that would be the implement and check or update one.

**Comment by TomBillingtonUK** [comment]
both next sprint

**Comment by mgoldson** [comment]
Yep, I just couldn't see a story in the backlog. Could be an easy one to miss

**Pull request merged**